internal/app/machined/pkg/system/runner/containerd/containerd_test.go

```diff
@@ -93,7 +94,9 @@ func (suite *ContainerdSuite) SetupSuite() {
 	suite.client, err = containerd.New(suite.containerdAddress)
 	suite.Require().NoError(err)
 
-	ctx := namespaces.WithNamespace(context.Background(), containerdNamespace)
+	suite.containerdNamespace = fmt.Sprintf("talostest%d%d", rand.Int63(), time.Now().Unix())
```

Comment:
containerd has weird validation rules for namespace name... e.g. `uuid` doesn't work

it should start with letters, parts after `-` should start with letter, etc.